README.md

```diff
@@ -96,7 +96,65 @@ Add Stumbles
             ]
           },
           type : "POST",
-          success : function(r) {
-            console.log(r);
+          success : function(req) {
+            console.log(req);
+          }
+        });
+
+Get Leaders
+----
+  Get the leaders for a given Country.
+
+* **URL**
+
+  https://leaderboard.services.mozilla.com/backend/leaders
+
+* **Method:**
+
+  `GET`
+
+*  **URL Params**
+
+  * country_id : `int`
+
+* **Data Params**
+
+  None
+
+* **Request Headers**
+
+  None
+
+* **Success Response:**
+
+  * **Code:** 200
+
+  JSON encoding
+
+        {
+          country_name: <str>,
+          leaders: [
+            {
+              name: <str>,
+              observations: <int>
+            }
+          ]
+        }
+
+* **Error Responses:**
+
+  None
+
+* **Sample Call:**
+
+        $.ajax({
+          url: "https://leaderboard.services.mozilla.com/backend/leaders",
+          dataType: "json",
+          data: {
+            country_id: 1,
```

Comment:
We probably want to add a `query_type` as the common case for stumbling events is going to be small regions - probably city scale.    You probably want to stuff the query_type into the URL so that you can do conditional get requests and cache queries.